lib/rex/post/postgresql/ui/console/command_dispatcher/modules.rb

```diff
@@ -0,0 +1,95 @@
+# -*- coding: binary -*-
+
+require 'pathname'
+
+module Rex
+  module Post
+    module PostgreSQL
+      module Ui
+        ###
+        #
+        # PostgreSQL client commands for running modules
+        #
+        ###
+        class Console::CommandDispatcher::Modules
+
+          include Rex::Post::PostgreSQL::Ui::Console::CommandDispatcher
+
+
+          #
+          # List of supported commands.
+          #
+          def commands
+            cmds = {
+              'run' => 'Run a module'
+            }
+
+            reqs = {}
+
+            filter_commands(cmds, reqs)
+          end
+
+          #
+          # Modules
+          #
+          def name
+            'Modules'
+          end
+
+          def cmd_run_help
+            print_line 'Usage: Modules'
```

Comment:
I think we'll want to beef up the help docs here a bit, as well as the original source from the SMB session